packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -192,14 +201,25 @@ export class WaferMap<
         return this.waferMapValidator.getValidity();
     }
 
-    public override connectedCallback(): void {
+    public override async connectedCallback(): Promise<void> {
         super.connectedCallback();
-        this.canvasContext = this.canvas.getContext('2d', {
-            willReadFrequently: true
-        })!;
         this.hoverHandler.connect();
         this.experimentalHoverHandler.connect();
         this.zoomHandler.connect();
+        const { matrixRenderer } = await createMatrixRenderer();
+        this.worker = matrixRenderer;
+        if (this.diesTable !== undefined) {
+            const offscreenOne = this.canvas.transferControlToOffscreen();
```

Comment:
Can we rename `offscreenOne` to `offscreenCanvas`?

Reply:
sure!